scenes/matchmaking/matchmaking.gd

```diff
@@ -1,58 +1,52 @@
 extends Control
 class_name Matchmaking
 
-enum STATES {
-	INIT
-}
-
-@onready var nInstrumentBox_Image = find_child("InstrumentBox/Image")
-@onready var nWaitingOpponent = find_child("WaitingOpponent")
-@onready var nWaitingOpponent_AnimationPlayer = nWaitingOpponent.find_child("AnimationPlayer")
-@onready var nSongSelection: SongSelection = find_child("SongSelection")
-@onready var nMatches: Matches = find_child("Matches")
-
-var game_match: NakamaRTAPI.Match
-var match_players_count = 0
+@onready var nInstrumentBox_Image = %InstrumentBox/Image
+@onready var nWaitingOpponent = %WaitingOpponent
+@onready var nWaitingOpponent_AnimationPlayer = %AnimationPlayer
+@onready var nSongSelection: SongSelection = %SongSelection
+@onready var nMatches: Matches = %Matches
```

Comment:
FWIW (I know it's not yours), this horrible naming convention can go. No capital letters and no prefixes should be used.

Reply:
As you said. it's not mine. I'd rather avoid extra conflicts with people submit code.